### PR TITLE
Added StandardCharsets.UTF_8 instead of Charset.forName("UTF-8")

### PR DESCRIPTION
**Pull request by abdullahcevik**

### CODE REVIEW

File: hazelcast/src/main/java/com/hazelcast/query/impl/getters/JsonPathCursor.java [hunk 1]
```diff
@@ -29,7 +30,7 @@
  */
 public class JsonPathCursor {
 
-    private static final Charset UTF8_CHARSET = Charset.forName("UTF8");
+    private static final Charset UTF8_CHARSET = StandardCharsets.UTF_8;
```
Comment (pivovarit) [inline hazelcast/src/main/java/com/hazelcast/query/impl/getters/JsonPathCursor.java:33]: I believe we don't need the static field at all if we can reuse the `StandardCharsets.UTF_8`

Comment (abdullahcevik) [reply hazelcast/src/main/java/com/hazelcast/query/impl/getters/JsonPathCursor.java:33]: I agree with you. If you want I can remove the static field.

Comment (pivovarit) [reply hazelcast/src/main/java/com/hazelcast/query/impl/getters/JsonPathCursor.java:33]: Yeah, that's the right way to go. There's no use in having a static field referencing another constant

Comment (abdullahcevik) [reply hazelcast/src/main/java/com/hazelcast/query/impl/getters/JsonPathCursor.java]: I removed redundant UTF8_CHARSET static field. 
https://github.com/hazelcast/hazelcast/pull/17741/commits/627d61ae28bb86249fee966c17878dc7a89e1895

**Review by pivovarit — APPROVED** [review]

**Comment by mmedenjak** [comment]
run-lab-run

**Review by mdumandag — APPROVED** [review]

**Comment by hz-highfive** [comment]
The job `Hazelcast-pr-builder` of your PR failed ([log](
https://j-artifacts.s3.amazonaws.com/Hazelcast-pr-builder/2793/build_log.txt), [artifacts](https://j-artifacts.s3.amazonaws.com/Hazelcast-pr-builder/2793/Hazelcast-pr-builder-2793.zip)). 
Through arcane magic we have determined that the following fragments from the build log may contain information about the problem.
<details>
<summary>Click to expand the log file</summary>
<pre>
--------------------------
---------SUMMARY----------
--------------------------
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-checkstyle-plugin:3.1.1:checkstyle (default) on project hazelcast: An error has occurred in Checkstyle report generation. Failed during checkstyle execution: There is 1 error reported by Checkstyle 8.29 with /home/jenkins/jenkins_slave/workspace/Hazelcast-pr-builder/checkstyle/checkstyle.xml ruleset. -> [Help 1]
--------------------------
---------ERRORS-----------
--------------------------
[ERROR] /home/jenkins/jenkins_slave/workspace/Hazelcast-pr-builder/hazelcast/src/main/java/com/hazelcast/query/impl/getters/JsonPathCursor.java:21:8: Unused import - java.nio.charset.Charset. [UnusedImports]
--------------------------
</details>
</pre>

**Comment by mdumandag** [comment]
@abdullahcevik It seems that there is an unused import here: https://github.com/hazelcast/hazelcast/pull/17741/files#diff-5f9e37f021878f2734bff96a587f4d8a21aa35ca37b440fa03c26c71b4b77624R21

Could you remove that?

**Comment by abdullahcevik** [comment]
> @abdullahcevik It seems that there is an unused import here: https://github.com/hazelcast/hazelcast/pull/17741/files#diff-5f9e37f021878f2734bff96a587f4d8a21aa35ca37b440fa03c26c71b4b77624R21
> 
> Could you remove that?

I removed unused import.

**Comment by mdumandag** [comment]
run-lab-run

**Comment by mmedenjak** [comment]
Merged, thank you for the contribution @abdullahcevik and the reviews @mdumandag @pivovarit !

**Pull request merged**